### ISSUE TITLE
How to setup the database tables?

### ISSUE THREAD

**Issue by xtroncode** [hacktx/nucleus#16]
Is there a command  for initial setup of the database or we have to create them manually ?

**Comment by rob0rt** [comment]
Ah, the undocumented part of the install. So you're going to need a MySQL install, create a database, and then run this sql script on that database:
https://github.com/hacktx/nucleus-vagrant/blob/master/provision/config/schema.sql
That _should_ get the DB setup like you need it. 

I'm still actively developing nucleus and haven't really had a v1 release, and there is a lot to do before it gets to that point (like, surviving HackTX, lol). Sorry about the lacking documentation and in some places fairly confusing pieces of code. I'm working to get everything in a more stable, easy to contribute to state, but have a lot of things I need to do before I get to that point.

**Comment by rob0rt** [comment]
You can take a look and see how the vagrant dev environment for Nucleus gets setup:
https://github.com/hacktx/nucleus-vagrant/blob/master/provision/setup.sh

It walks through the steps in order and is fairly well commented.

**Comment by shaktigoap** [comment]
Hey @BobertForever! We are trying to use Nucleus for our Hackathon. @xtroncode Wanted to ask some questions about the features being supported.

**Comment by rob0rt** [comment]
sorry for the belated response. I saw your email @shaktigoap I've just been very consumed with building out the features we need as we need them, and my birthday was this weekend so I was... preoccupied. There's currently no database setup scripts or configurations; the only thing I've prepared is a development environment for Nucleus ([nucleus-vagrant](https://github.com/hacktx/nucleus-vagrant)), which contains the DB schema. I have many plans to build out documentation and reusability once our event is in the past.

If you have any specific questions about features I can try my best to answer them.
